README.md

```diff
@@ -201,6 +202,25 @@ out of the box with EmberJS.
 - Type: `boolean`
 - Default: `true`
 
+### errorStackInResponse
+Along handleErrors, When true, this option will send the error stack if available within the error
+reponse. It will be stored under the `source.stack` key.
+
+**Be careful, this option should never be enabled in production environment. It can propagate some
```

Comment:
Please be careful, this option should never be enabled in a production environment. Doing so can expose sensitive data.